egs/lre07/v2/run.sh

```diff
@@ -0,0 +1,244 @@
+#!/bin/bash
+# Copyright  2016   GoVivace Inc. (Author: Mousmita Sarma) 
+#                   
+# Apache 2.0.
+#
+# This script runs the NIST 2007 General Language Recognition Closed-Set
+# evaluation.
+# This example script shows how to replace the GMM-UBM
+# with a DNN trained for ASR. 
+
+. cmd.sh
+. path.sh
+set -e
+
+mfccdir=`pwd`/mfcc
+vaddir=`pwd`/mfcc
+languages=local/general_lr_closed_set_langs.txt
+nnet=exp/nnet2_online/nnet_ms_a/final.mdl
+
+# Train a DNN on about 1800 hours of the english portion of Fisher.
+local/dnn/train_dnn.sh
+
+
+# Training data sources
+local/make_sre_2008_train.pl /home/mousmita/DATA/LDC2011S05 data
+local/make_callfriend.pl /home/mousmita/DATA/LDC96S60 vietnamese data
+local/make_callfriend.pl /home/mousmita/DATA/LDC96S59 tamil data
+local/make_callfriend.pl /home/mousmita/DATA/LDC96S53 japanese data
+local/make_callfriend.pl /home/mousmita/DATA/LDC96S52 hindi data
+local/make_callfriend.pl /home/mousmita/DATA/LDC96S51 german data
+local/make_callfriend.pl /home/mousmita/DATA/LDC96S50 farsi data
+local/make_callfriend.pl /home/mousmita/DATA/LDC96S48 french data
+local/make_callfriend.pl /home/mousmita/DATA/LDC96S49 arabic.standard data
+local/make_callfriend.pl /home/mousmita/DATA/LDC96S54 korean data
+local/make_callfriend.pl /home/mousmita/DATA/LDC96S55 chinese.mandarin.mainland data
+local/make_callfriend.pl /home/mousmita/DATA/LDC96S56 chinese.mandarin.taiwan data
+local/make_callfriend.pl /home/mousmita/DATA/LDC96S57 spanish.caribbean data
+local/make_callfriend.pl /home/mousmita/DATA/LDC96S58 spanish.noncaribbean data
+local/make_lre96.pl /home/mousmita/DATA/lid96e1 data
+local/make_lre03.pl /home/mousmita/DATA/LDC2006S31 data
+local/make_lre05.pl /home/mousmita/DATA/LDC2008S05 data
+local/make_lre07_train.pl /home/mousmita/DATA/LDC2009S05/package/lid07tr1 data
+local/make_lre09.pl /home/mousmita/DATA/LRE2009/eval data
```

Comment:
Could you change the paths to the ones used in https://github.com/s-mousmita/kaldi/blob/62e6e06da7aaad5e7d34e4aa99fc8f7823a1a3c3/egs/lre07/v1/run.sh ? I think that would be a little more consistent.